service/siteService.js

```diff
@@ -1,6 +1,16 @@
 
-const airtableClient = require('./airtableClient');
+const airtableClient = require('../airtableClient');
 const cacheService = require('./cacheService');
+const PublicTransitEnum = require('../domain/PublicTransitEnum')
+
+const LIGHTRAIL_ICON = "tram"
```

Comment:
Was thinking about extracting these out into constants file? But the

Reply:
The comment cut off, but I wouldn't worry about it yet.  It's still a pretty small file.

Reply:
Oh haha yeah, I don't even remember what I was trying to type whoops! but sounds good.